server/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -301,59 +267,47 @@
             + "The value is stored in {{%s}} by default unless a template name is provided; if so, it is then stored in the template's {{%s}} file.\n"
             + "If the value is empty (''), then the property is unset.\n"
             + "This option is implicit if no '--get' or '--get-regexp' option is used and there are exactly two parameters (key value).",
-            ConfigurationGenerator.NUXEO_CONF, ConfigurationGenerator.NUXEO_DEFAULT_CONF);
+            NUXEO_CONF, ConfigurationGenerator.NUXEO_DEFAULT_CONF);
```

Comment:
Though `ConfigurationGenerator.NUXEO_DEFAULT_CONF` is only used once, what about using static import for consistency with `NUXEO_CONF`?